addons/account/models/account_move.py

```diff
@@ -974,8 +974,9 @@ def compute_writeoff_counterpart_vals(values):
                     raise UserError(_("Either pass both debit and credit or none."))
                 if 'date' not in vals:
                     vals['date'] = self._context.get('date_p') or fields.Date.today()
-                    if vals['date'] < date:
-                        date = vals['date']
+                vals['date'] = fields.Date.to_date(vals['date'])
+                if vals['date'] and vals['date'] < date:
```

Comment:
Is this the intended behavior? It means that if I choose to create a writeoff with a future date, the date I choose in the view won't be taken into account and instead it will be today's date.

Reply:
What it is doing it that it chooses the earliest date from all the lines to put it as the date of the move here https://github.com/odoo/odoo/blob/17594b8a51a5a80feb5a22f589b115eb6356fdfe/addons/account/models/account_move.py#L1012-L1017
I guess that it is correct that an account.move can not be written in the future ?

Reply:
ok let's leave it like this then :)